packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -741,6 +741,18 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
     UserScrollNotification(metrics: copyWith(), context: context.notificationContext, direction: direction).dispatch(context.notificationContext);
   }
 
+  /// Provides a heuristic to determine if expensive frame-bound tasks should be
+  /// deferred.
+  ///
+  /// The actual work of this is delegated to the [physics] via
+  /// [ScrollPhysics.recommendDeferredScrolling] called with the current
+  /// [activity]'s [ScrollActivity.velocity].
+  bool recommendDeferredLoading() {
```

Comment:
could be a getter? not sure how i feel about it

Reply:
The other simiilarly named ones are methods - just seems consistent this way to me.